### ISSUE TITLE
SVG console error

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/calculus-grapher#338]
**Test device**
MacBook Air M1 chip

**Operating System**
14.1.2

**Browser**
Safari 17.1.2

**Problem description**
When testing https://phet-dev.colorado.edu/html/calculus-grapher/1.0.6-rc.1/phet/calculus-grapher_all_phet.html for https://github.com/phetsims/qa/issues/1017, some items in the nav bar disappeared (title, PhET menu, preferences menu) and the screen icons were flipped and upside down. 
I haven't been able to reproduce this yet, but there is an SVG error in the console.

**Visuals**
<img width="821" alt="Screenshot 2023-12-22 at 10 51 42 AM" src="https://github.com/phetsims/calculus-grapher/assets/87318828/fe7aac8e-2f1d-484d-8064-c7538d7fc112">
<img width="511" alt="Screenshot 2023-12-22 at 10 51 57 AM" src="https://github.com/phetsims/calculus-grapher/assets/87318828/b4a07541-2621-4d41-85bf-cc60d75e5a7c">

**Comment by pixelzoom** [comment]
Yikes.

There have been no changes to this sim, and the problem is in joist.  This must be related to SVG work that @jonathanolson has been doing.  So unassigning myself.

**Comment by jonathanolson** [comment]
@amanda-phet and @marlitas, is it possible there's a bad SVG file getting generated here?

**Comment by jonathanolson** [comment]
Hmm, I'm unable to reproduce, Apple M1 Pro chip, macOS 14.2, Safari 17.2.

**Comment by amanda-phet** [comment]
I am not sure what svgs are in the navbar. Are the preferences and sound icons svgs?

**Comment by zepumph** [comment]
I'm on Windows and was not able to reproduce. I would most likely not raise the priority on this until we are able to reproduce it.

**Comment by Nancy-Salpepi** [comment]
I tried for a while earlier today and couldn't reproduce it.

**Comment by pixelzoom** [comment]
@zepumph said:
> ... I would most likely not raise the priority on this until we are able to reproduce it.

But it's being reported in an RC, so will presumably be published like this if we don't raise the priority.  That said, I have not been able to reproduce this either in Safari 17.1.2.   I am running macOS 14.2.1, not 14.1.2.

**Comment by zepumph** [comment]
Yes, it is a challenging concept for PhET at large that there are likely hundreds of bugs in our published sims. We care a lot about the quality of our product! Mostly they are really hard to find and not that destructive to the UX, otherwise we probably would have been able to find and solve it earlier. We are a small team, and need to always determine how best we should spend our time. If 5 people (including the original reporter of the bug) have not been able to figure out how to reproduce the bug, perhaps spending a combined total of an hour on this issue, then likely we should focus on higher priority work.

**Comment by marlitas** [comment]
> is it possible there's a bad SVG file getting generated here?

We haven't generated any SVGs for common code... Everything has been isolated to sim specific repos. Nothing that I would imagine Calculus Grapher is dependent on.

**Comment by pixelzoom** [comment]
This issue has been open for a long time. CT shows no sign of the problem.  Comments above indicate that no one has been able to reproduce this.   So I'm going to close.
